train.py

```diff
@@ -218,6 +219,11 @@ def loss_fn(pred, labels):
     # apply fp8 linear module swap
     if job_config.training.fp8_linear:
         build_fp8_linear(whole_model, job_config)
+    else:
```

Comment:
can remove this to favor simplicity if it is a no-op flag when `fp8_linear=False`

Reply:
removed `ValueError` on `enable_fp8_linear=False`